cms/djangoapps/contentstore/views/item.py

```diff
@@ -333,6 +336,32 @@ def xblock_outline_handler(request, usage_key_string):
         return Http404
 
 
+# pylint: disable=unused-argument
```

Comment:
I think @brianhw once told me that prefixing parameters with `_` will make pylint disable the unused-argument warning.